pkg/announce/announce.go

```diff
@@ -143,3 +154,33 @@ func create(workDir, subject, message string) error {
 
 	return nil
 }
+
+// getGoVersion runs kube-cross container and go version inside it.
+// We're running kube-cross container because it's not guaranteed that
+// k8s-cloud-builder container will be running the same Go version as
+// the kube-cross container used to build the release.
+func getGoVersion(tag string) (string, error) {
+	semver, err := util.TagStringToSemver(tag)
+	if err != nil {
+		return "", errors.Wrap(err, "parse version tag")
+	}
+
+	releaseBranch := fmt.Sprintf("release-%d.%d", semver.Major, semver.Minor)
+
+	kubecrossVer, err := release.GetKubecrossVersion(releaseBranch)
```

Comment:
I'm considering to include the master branch here as well, such as:

```suggestion
	kubecrossVer, err := release.GetKubecrossVersion(releaseBranch, "master")
```

(I think there might be something like `git.DefaultBranch` or something like that, instead of hard-coded `master`)

I think that it might happen that we cut a release that still doesn't have the kube-cross variant (that should be all releases before and including rc.0). In that case, this function would fail, but if we add the master branch, it would have a fall-back.

Reply:
Yep, we use `git.DefaultBranch` everywhere (I see you used it, nice :)